t/02mysql-parser.t

```diff
@@ -502,6 +502,19 @@ BEGIN {
     is( join(',', $t2c2->reference_fields), 'id', 'To field "id"' );
 }
 
+# Tests for CREATE VIEW statements that contain a column list
+# after the view name
+{
+    my $tr = SQL::Translator->new();
+    my $data = parse($tr,
+        q[
+            CREATE
+              VIEW view_foo (id, name) AS
+                SELECT id, name FROM thing;
+        ]
+    ) or die $tr->error;
```

Comment:
Please also test that the view is in fact correctly parsed and that the column names after the view name take precedence over the ones in the `SELECT` list.